### PR TITLE
docs(use-linea): add patch wallet and linea domains tutorials

### PR DESCRIPTION
**Pull request by corbinpage**
# Pull request checklist

Use the following template to make sure your PR fits the ConsenSys documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://docs-template.consensys.net/getting-started/style-guide).
- [x] You've previewed your changes locally.

## Describe the change

Added the Patch wallet and LineaL2.domains tutorials

## Issue fixed

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've previewed your changes on Vercel below.

### CODE REVIEW

File: docs/use-linea/explore/use-patch.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Use Patch Wallet
+sidebar_position: 1
+---
+
+# Patch Wallet
+[Patch Wallet](https://linea.patchwallet.com/) is a "zero onboarding" wallet attached to every Twitter account, email, and telephone number. There's no app to download, no gas, or even sign-up. Every Twitter user has a wallet automatically and can start receiving tokens and NFTs!
+
+In this guide, we'll walk you through how to access your Patch wallet connected to your Twitter, mint an NFT, and send it to *any* Twitter user...[Elon Musk](https://linea.patchwallet.com/twitter:elonmusk), [Cristiano Ronaldo](https://linea.patchwallet.com/twitter:cristiano), [Narendra Modi](https://linea.patchwallet.com/twitter:narendramodi) or anyone else!
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-patch.md:9]: Screenshot 2023-06-13 at 6 06 40 PM

Hi can you add the failing words to `project-words.txt` (in alphabetical order at the top - I'm aware the bottom words are all out of order)
And run `npm run lint` to make sure they all pass

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]

File: docs/use-linea/explore/use-patch.md (hunk 1 above)
Comment (corbinpage) [reply docs/use-linea/explore/use-patch.md:9]: Ok, added @emilyJLin95 👍

Let me know if you need any other changes.

File: project-words.txt [hunk 2]
```diff
@@ -699,4 +699,9 @@ redcircle
 bluecircle
 tatarot
 Tatarot
-Scry
\ No newline at end of file
+Scry
```
Comment (emilyJLin95) [inline project-words.txt:702]: this is causing merge conflicts
add them in alphabetical order at the top to avoid

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]

File: project-words.txt (hunk 2 above)
Comment (emilyJLin95) [reply project-words.txt:702]: @corbinpage

Comment (emilyJLin95) [reply project-words.txt:702]: nvm we need these changes in an hour so I just copied them and merged in my own PR

Comment (corbinpage) [reply project-words.txt:702]: Thanks Emily! Apreciate the help!

**Pull request closed without merging**